README.md

```
@@ -1,81 +1,278 @@
-# Turborepo starter
+<div align="center">
+  <img src="https://github.com/user-attachments/assets/38a66efd-c2c1-4b87-8a5a-497605a72ad3" width="100px" height="100px" />
+  <h1>Codeit Resources</h1>
+  
+[![GitHub issues](https://img.shields.io/github/issues/codeit-internship-group-b/codeit-resources?color=red)]()
```

Markdownlint (0.37.0)

No empty links

(MD042, no-empty-links)